### PR TITLE
ARS-433 Check registered details page

### PR DESCRIPTION
**Pull request by RawToast**
Does **not** include navigation, as the pages do not exist

スクリーンショット 0005-01-09 14 43 53

### CODE REVIEW

File: app/views/components/Warning.scala.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(content: Html, classes: String = "govuk-warning-text__text", id: Option[String] = None)
```
Comment (C-Thomp) [inline app/views/components/Warning.scala.html:19]: Nice, shall we default an id here? Something like `warning-text` or `notice`?

**Review by C-Thomp — APPROVED** [review]
Approved with minor suggestion 👍

File: app/views/components/Warning.scala.html (hunk 1 above)
Comment (RawToast) [reply app/views/components/Warning.scala.html]: https://github.com/hmrc/advance-valuation-rulings-frontend/pull/17/commits/3201e0e4b93db1ee3c25c4e0415dabb5c647344b

**Review by C-Thomp — APPROVED** [review]
🎉

**Pull request merged**